source/robot_model/include/robot_model/Model.hpp

```diff
@@ -6,13 +6,22 @@
 #include <OsqpEigen/OsqpEigen.h>
 #include <pinocchio/parsers/urdf.hpp>
 #include <pinocchio/multibody/data.hpp>
+<<<<<<< HEAD
+#include <pinocchio/algorithm/joint-configuration.hpp>
+#include <state_representation/Parameters/Parameter.hpp>
+#include <state_representation/Parameters/ParameterInterface.hpp>
+#include <state_representation/Robot/Jacobian.hpp>
+#include <state_representation/Robot/JointState.hpp>
+#include <state_representation/Space/Cartesian/CartesianState.hpp>
+=======
```

Comment:
Oh oh careful you have a conflict and you have added conflicting lines here!

Reply:
ops